java/src/jmri/jmrit/beantable/BlockTableAction.java

```diff
@@ -981,12 +981,12 @@ void okPressed(ActionEvent e) {
                 return;
             }
         }
-        String user = userName.getText().trim(); // N11N
+        String user = userName.getText().trim();
```

Comment:
Please don't use `trim()` here.  Instead, pass the name through the `normalizeSystemName(@Nonnull String inputName)` in the manager. (Multiple places)